packages/autorest.typescript/package.json

```diff
@@ -80,7 +80,7 @@
   },
   "devDependencies": {
     "@azure-tools/test-recorder": "^3.0.0",
-    "@azure/abort-controller": "^1.0.1",
+    "@azure/abort-controller": "^2.1.2",
```

Comment:
Could you update other places also?
- https://github.com/Azure/autorest.typescript/blob/main/packages/rlc-common/src/metadata/packageJson/azurePackageCommon.ts#L45-L50
- https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-ts/package.json#L63

Reply:
Will do!

Reply:
Done